packages/ra-core/src/controller/useEditController.ts

```diff
@@ -146,8 +147,11 @@ const useEditController = (props: EditProps): EditControllerProps => {
         resource,
         basePath,
         record,
+        redirect: DefaultRedirect,
```

Comment:
I don't understand, the redirect target should be a  parameter of the controller, not something it returns..

Reply:
It's only the default. The "real" redirect can be specified on the form itself.